packages/storybook-react/src/stories/CodeBlock.stories.tsx

```diff
@@ -31,7 +31,7 @@ const meta = {
     docs: {
       page: () => (
         <>
-          <Description markdown={readme} />
```

Comment:
CodeBlock geeft bij mij een foutmelding bij de rendering van parameters.docs.page: e.background is undefined